src/SideciTest.php

```diff
@@ -0,0 +1,20 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: Yoshihiro
+ * Date: 4/22/16
+ * Time: 11:24 PM
+ */
+
+namespace OhSky\LineTrialBot;
```

Comment:
__[code_sniffer]__
* There must be one blank line after the namespace declaration